src/azure-cli/azure/cli/command_modules/vm/tests/hybrid_2018_03_01/test_vm_commands.py

```diff
@@ -1497,7 +1499,7 @@ def test_vmss_accelerated_networking(self, resource_group):
         self.kwargs.update({
             'vmss': 'vmss1'
         })
-        self.cmd("vmss create -n {vmss} -g {rg} --vm-sku Standard_DS4_v2 --image Win2022Datacenter --admin-username clittester --admin-password Test12345678!!! --accelerated-networking --instance-count 1")
+        self.cmd("vmss create -n {vmss} -g {rg} --vm-sku Standard_DS4_v2 --image Win2012R2Datacenter --admin-username clittester --admin-password Test12345678!!! --accelerated-networking --instance-count 1")
```

Comment:
fix
![Win2022Datacenter](https://user-images.githubusercontent.com/18628534/151328115-7edc5dcc-d04e-4f4a-99cc-6fd64b89a339.png)

Reply:
Using image a is by design, PR link: https://github.com/Azure/azure-cli/pull/20701, I think we need to investigate the reason why this image can no longer be found